### PR TITLE
make bpfink more fail-tolerant during startup & runtime

### PR DESCRIPTION
**Pull request by nloginov**
1. Bad consumer initialization during startup does not lead to crash anymore
2. BPF kernel metrics file bad closing doesn't lead to crash anymore
3. Links to socket files are not monitored anymore
4. Removed some unreachable/redundant code

### CODE REVIEW

File: pkg/graphite.go [hunk 1]
```diff
@@ -150,7 +149,7 @@ func (m *Metrics) fetchBPFMetrics() (map[string]bpfMetrics, error) {
 	}
 	defer func() {
 		if err := file.Close(); err != nil {
-			log.Fatal(err)
```
Comment (tehmaze) [inline pkg/graphite.go:153]: Should probably read `log.Error`?

File: pkg/graphite.go [hunk 2]
```diff
@@ -209,7 +208,7 @@ func (m *Metrics) fetchBPFMetrics() (map[string]bpfMetrics, error) {
 	}
 
 	if err := scanner.Err(); err != nil {
-		log.Fatal(err)
```
Comment (tehmaze) [inline pkg/graphite.go:212]: Should probably read log.Error?

**Review by tehmaze — CHANGES_REQUESTED** [review]
Please check the logging levels.

File: pkg/graphite.go (hunk 1 above)
Comment (nloginov) [reply pkg/graphite.go:153]: you don't believe it, but standart golang log library doesn't know anything about error logging level, it allows either fatal or print:
https://golang.org/pkg/log/

**Review by tehmaze — APPROVED** [review]

**Pull request merged**